pandas/_libs/lib.pyx

```diff
@@ -120,13 +120,9 @@ def memory_usage_of_objects(arr: object[:]) -> int64_t:
     """
     i: Py_ssize_t
     n: Py_ssize_t
-    size: int64_t
 
-    size = 0
     n = len(arr)
-    for i in range(n):
-        size += arr[i].__sizeof__()
-    return size
+    return sum(arr[i].__sizeof__() for i in range(n))
```

Comment:
have you profiled this? i expect this is slower

Reply:
(and if im wrong, then how about going one step further and just do `sum(x.__sizeof__() for x in arr)`?)

Reply:
I have not profiled this, as ```asv``` requires conda to be installed, and I use ```venv```.

Reply:
1) `%timeit` profiling is a good alternative, especially for small things like this
2) i use `asv continuous -E virtualenv`